docs/source/tutorial-ssh.rst

```diff
@@ -0,0 +1,178 @@
+.. _tutorial-ssh:
+
+Tutorial: SSH Resource Workflows
+*********************************
+
+This tutorial walks you through ReproMan workflows using SSH resources, from simple command execution to complex data analysis.
+We'll start with a basic hello-world example, then progress to processing neuroimaging data.
+
+This tutorial demonstrates ReproMan's power in creating reproducible, traceable computational workflows across SSH-accessible computing environments.
+
+Overview
+========
+
+We'll cover two workflows:
+
+**Part 1: Hello World Example**
+1. Create a ReproMan SSH resource  
+2. Execute a simple command remotely
+3. Fetch and examine results
+
+**Part 2: Dataset Analysis Example**
+1. Set up a DataLad dataset with input data
+2. Execute MRIQC quality control analysis remotely  
+3. Collect and examine results with full provenance
+
+Prerequisites
+=============
+
+- ReproMan installed (``pip install reproman``) 
+- Access to a remote server via SSH
+- For Part 2: DataLad support (``pip install 'reproman[full]'``)
```

Comment:
TODO make it clear that install is on local machine, reproman does not need to be installed on resource, but datalad does)